pkg/client/postgres/tasks.go

```diff
@@ -95,24 +136,69 @@ func (db *PostgresDB) ScheduleBackup(schedule string) error {
 	return db.client.Apply(db.Namespace, job)
 }
 
-func (db *PostgresDB) ListBackups() ([]string, error) {
-	var backups []string
-	return backups, nil
+func (db *PostgresDB) ListBackups(s3Bucket string) error {
+	opClusterEnv := *db.opClusterEnv
+
+	var backupS3Bucket string
+	if s3Bucket != "" {
+		backupS3Bucket = s3Bucket
+	} else {
+		backupS3Bucket = db.backupBucket
+	}
+
+	job := kommons.
+		Deployment("list-backups-"+db.Name+"-"+utils.ShortTimestamp(), string(opClusterEnv["BACKUP_IMAGE"])).
+		EnvVarFromSecret("PGPASSWORD", db.Secret, "password").
+		EnvVars(map[string]string{
+			"AWS_ACCESS_KEY_ID":     string(opClusterEnv["AWS_ACCESS_KEY_ID"]),
```

Comment:
Cant we reuse `backup-postgres-{{ .metadata.name }}-config` here?

Reply:
We are not copying `AWS_ACCESS_KEY_ID` (and some other configurations) from the operatorClusterEnv secret to `backup-postgres-{{ .metadata.name }}-config` secret. We can do that and that might simplify the go code here as we don't have to pull values from 2 different secrets. What do you think?

Reply:
Yes that could work, and then we can just use envFrom and not env to import all the secrets

Reply:
This is done but also depends on this PR: https://github.com/flanksource/kommons/pull/32 (add support to use `envFrom`)